### DISCUSSION TITLE
Hindi Language support

### DISCUSSION THREAD

**Discussion by jaggadas** [explosion/spaCy#10989] [Language Support]
Is Hindi language not supported on spacy 2.xx?

**Comment by polm** [comment]
Based on [the v2 models page](https://v2.spacy.io/models) it appears to not be supported. 

v3 has been out for over a year, and if you're starting a new project I'd strongly recommend it. Can you clarify why you'd like to use v2?

**Reply by jaggadas** [reply]
I am trying to use spacy model on android through a library called chaquopy which only supports till spacy 2.3.2

**Reply by polm** [reply]
Thanks for the clarification, I can see how that limits your choices. 

[Looking at the source](https://github.com/explosion/spaCy/tree/v2.3.2/spacy/lang/hi), it looks like v2.3.2 did have tokenization support for Hindi, so you should be able to use that to train your own model.
